docs/features/translation_memory.rst

```diff
@@ -77,9 +77,13 @@ Pootle supports two types of Elasticsearch-based TMs:
 - **External TMs**: (several) are populated from translation files specifically
   provided by the server admins, and are not automatically updated.
 
-
```

Comment:
Please don't remove this blank line as it makes the text more readable when editing it.